File: TFTP/Server/src/main/java/impl/tftp/CommandError.java
package bgu.spl171.net.impl.tftp;

public class CommandError extends Command {

	private short errorCode;
	private String errorMsg;
	
	public CommandError(short errorCode, String errorMsg) {
		super((short) 5);
		this.errorCode = errorCode;
		this.errorMsg = errorMsg;
	}
	
	public short getErrorCode(){
		return errorCode;
	}
	
	public String getErrorMsg(){
		return errorMsg;
	}

}

File: TFTP/Client/src/CommandError.cpp
#include "CommandError.h"

using namespace std;

CommandError::CommandError(short errorCode, std::string errorMsg) : Command((short)5), errorCode(errorCode), errorMsg(errorMsg){}

string CommandError::getErrorMsg(){
	return errorMsg;
}

short CommandError::getErrorCode(){
	return errorCode;
}

CommandError::~CommandError(){}
